components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/internal/UserStoreConfigListenersHolder.java

```diff
@@ -85,6 +87,30 @@ public void setAllowedUserstores(Set<String> allowedUserstores) {
         this.allowedUserstores = allowedUserstores;
     }
 
+    /**
+     * Get all user store attribute mappings.
+     *
+     * @return UserStoreAttributeMappings.
+     */
+    public UserStoreAttributeMappings getUserStoreAttributeMappings() {
+
+        if (userStoreAttributeMappings == null) {
+            // Read user store attributes mappings from files.
+            AttributeMappingBuilder.getInstance().build();
```

Comment:
I am not getting the logic here.. Once this is built, it's not assigned to anything.. just returns userStoreAttributeMappings which is totally not related to the one you built

Reply:
AttributeMappingBuilder is a seperate builder class. From it, we build userstore attribute mappings and set value to `userStoreAttributeMappings` variable in this holder class.

Reply:
Then better to update the comment as **// Read user store attributes mappings from files and assigns it to userStoreAttributeMappings**

Reply:
We removed this class as it in not needed. This logic is moved to component class.Fixed https://github.com/wso2/carbon-identity-framework/pull/3781